data/schema/types/ship.js

```diff
@@ -0,0 +1,12 @@
+export default `
+type Engine {
```

Comment:
I wasn't aware of a simulator object.  I'll do a better job at making sure I look through all the other files first.